mmselfsup/models/algorithms/mae_classification.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from ..builder import ALGORITHMS, build_backbone, build_head
+from ..utils import Mixup
+from .base import BaseModel
+
+
+@ALGORITHMS.register_module()
+class MAEClassification(BaseModel):
```

Comment:
The main modification is about use mixup? Consider merge mae_classification with existing classification algorithm by supporting mixup in existing algorithm. This backbone seems to be not specially designed for MAE for now.